templates/home/contact.html.twig

```diff
@@ -5,27 +5,37 @@
 {% block body %}
 <div class = "container-fluid">
```

Comment:
en global, ton fichier est mal identé (normalement l'indentaiton doit etre de 4 espaces, et au dela de ça tes div ne sont pas tjs bien alignés, quand tu as ton for, le code à l'intérieur doit etre indenté, etc